### ISSUE TITLE
Can not get the same test accuracy as yours

### ISSUE THREAD

**Issue by LitterQ** [taki0112/TripleGAN-Tensorflow#3]
I can't get the same test accuracy as yours with --n 4000 --epoch 1000 --batch_size 20 --unlabel_batch_size 250 --z_dim 100, my test accuracy is only about 66% ? What may be the problem?

**Comment by fYYw** [comment]
Cannot reproduce the reported accuracy.  Also, the following code looks weird to me.
 `max_c = tf.cast(tf.argmax(Y_c, axis=1), tf.float32)`
`c_loss_dis = tf.reduce_mean(max_c * tf.nn.softmax_cross_entropy_with_logits(logits=D_cla_logits, labels=tf.ones_like(D_cla)))`

I changed the code to
`max_c = tf.max(Y_c, axis=1)`
`c_loss_dis = tf.reduce_mean(max_c * sigmoid_cross_entropy_with_logits(logits=D_cla_logits, labels=tf.ones_like(D_cla)))`
The best acc I got so far is 69%. 

The original theano implementation uses mean_batch_normalization, ZCA_whitenning, weight_normalization, and casts Y_c into one-hot encoding instead of distributions. May these differences lead to a 20% improvement of acc?

**Comment by leao1995** [comment]
Have any of you figured out the reason for low accuracy?

**Comment by roomo7time** [comment]
I got 66% accuracy.
